Maths/FindTrianglePerimeter.js

```diff
@@ -0,0 +1,9 @@
+/*
+* A polygon with three edges and three vertices is called a triangle. It is one of the fundamental geometric shapes.
+* The total length of a triangle's sides makes up the perimeter.
+* Algorithm to find the perimeter of a triangle
+*/
+function findTrianglePerimeter (x, y, z) {
+  return x + y + z
```

Comment:
This is too trivial for my taste. If `x`, `y`, `z` were points there would be some value in this, but just adding three side lengths? That's too trivial...

Reply:
Okay, maybe I will try to contribute to other useful algorithms that is not trivial perhaps...